go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/karpenter
 
-go 1.22.4
+go 1.22.2
```

Comment:
Would you mind to share why?

Reply:
Just a test attempt. Some downstream users of this package were running into some painpoints around upgrading to recently released go versions, so we were trying to see if this is possible. I'm not fully convinced that we should do this yet, though.

Reply:
thanks for sharing